sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/RedundantJumpStatement.cs

```diff
@@ -168,5 +168,28 @@ void RedundantJumpInTryCatch2()
                 Console.WriteLine();
             }
         }
+
+        void CSharp8_StaticLocalFunctions()
+        {
+            static void Compute(int a, out int b)
+            {
+                b = a;
+                return;     // Noncompliant
+            }
+        }
+    }
+
+    public interface IWithDefaultImplementation
+    {
+        decimal Count { get; set; }
+        decimal Price { get; set; }
+
+        //Default interface methods
+        void Reset()
+        {
+            Price = 0;
+            Count = 0;
+            return;     // Noncompliant
```

Comment:
For completeness sake, could we add a compliant example for both static local functions and default interface implementation